doc/source/geotransformations.md

```diff
@@ -0,0 +1,204 @@
+---
+file_format: mystnb
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: geoutils-env
+  language: python
+  name: geoutils
+---
+(geotransformations)=
+# Transformations
+
+In GeoUtils, **for all geospatial data objects, georeferenced transformations are exposed through the same functions** 
+{func}`~geoutils.Raster.reproject`, {func}`~geoutils.Raster.crop` and {func}`~geoutils.Raster.shift`. Additionally, 
+for convenience and consistency during analysis, most operations can be passed a {class}`~geoutils.Raster` or 
+{class}`~geoutils.Vector` as a reference to match.
+In that case, no other argument is necessary. For more details, see {ref}`core-match-ref`.
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# To get a good resolution for displayed figures
+from matplotlib import pyplot
+pyplot.rcParams['figure.dpi'] = 600
+pyplot.rcParams['savefig.dpi'] = 600
+pyplot.rcParams['font.size'] = 9
+```
+
+## Reproject
+
+{func}`geoutils.Raster.reproject` or {func}`geoutils.Vector.reproject`.
+
+Reprojections **transform geospatial data from one CRS to another**.
+
+For vectors, the transformation of geometry points is exact. However, in the case of rasters, the projected points 
+do not necessarily fall on a regular grid and require re-gridding by a 2D resampling algorithm, which results in a slight 
+loss of information (value interpolation, propagation of nodata).
+
+For rasters, it can be useful to use {func}`~geoutils.Raster.reproject` in the same CRS simply for re-gridding, 
+for instance when downsampling to a new resolution {attr}`~geoutils.Raster.res`.
+
+```{tip}
+Due to the loss information when re-gridding, it is important to **minimize the number of reprojections during the 
```

Comment:
"loss information" -> either "loss of information" or "information loss"

Reply:
Done